timescaledb/overview/page-index/page-index.js

```diff
@@ -79,7 +79,15 @@ module.exports = [
             tags: ['install', 'timescaledb', 'tsc', 'mst', 'self-hosted'],
             keywords: ['install', 'TimescaleDB', 'Timescale Forge', 'Timescale Cloud', 'self-hosted'],
             excerpt: "Deployment options for TimescaleDB",
+          children: [
+          {
+            title: "TimescaleDB on Kubernetes",
+            tags: ['install', 'timescaledb', 'self-hosted', 'Kubernetes', 'Helm Charts'],
+            keywords: ['install', 'TimescaleDB', 'self-hosted', 'Kubernetes'],
+            excerpt: "Deployment options for TimescaleDB",
```

Comment:
There is no `href` property so as far as I know there's no way for Docs to actually locate this page.